orquesta/statuses.py

```diff
@@ -58,7 +60,8 @@
     SCHEDULED,
     DELAYED,
     RUNNING,
-    RESUMING
+    RESUMING,
+    RERUNNING
```

Comment:
Remove if these statuses are no longer needed.  If that's the case, rebase the PR to not have a commit to add/delete them.